### DISCUSSION TITLE
Add more Netflix profiles in Conventions

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#264] [Ideas]
English and Spanish have been implemented, but not the other languages listed [here](https://partnerhelp.netflixstudios.com/hc/en-us/sections/203480497-Timed-Text-Style-Guides).

Using Tero Subtitler alpha - Mac 10.11.6.

**Comment by URUWorks** [comment]
v1.0.1.7

**Comment by chenlung** [comment]
Only a few have been added.
